src/components/NavBar/NavItem.tsx

```diff
@@ -18,39 +18,37 @@ type NavItemProps = {
 
 const styles = {
   navItem: {
-    borderRadius: "30px",
-    width: "fit-content",
-    paddingLeft: "12px",
-    paddingRight: "12px",
-    transitionDuration: "0.2s",
-    "&.Mui-selected": { backgroundColor: "transparent" },
+    "&.Mui-selected": {
+      ":hover": {
+        backgroundColor: "primary.light",
+      },
+      backgroundColor: "transparent",
+    },
     ":hover": {
       backgroundColor: "primary.light",
     },
+    borderRadius: 10,
+    paddingX: 1.5,
+    transitionDuration: "0.25s",
```

Comment:
for consistency in my suggestion 
```suggestion
    transition: "0.25s ease",
```